vulture/core.py

```diff
@@ -121,9 +127,12 @@ def __hash__(self):
 
 class Vulture(ast.NodeVisitor):
     """Find dead code."""
-    def __init__(self, exclude=None, verbose=False, sort_by_size=False):
+    def __init__(self, exclude=None, verbose=False, sort_by_size=False,
+                 min_confidence=0.5):
         self.exclude = []
         self.sort_by_size = sort_by_size
```

Comment:
if min_confidence < 0 or min_confidence > 1:
        raise ValueError('Value of min_confidence must be between 0 and 1.')